src/app/core/components/storage-dialog/storage-dialog.component.ts

```diff
@@ -1,10 +1,17 @@
 import { ActivatedRoute, ParamMap } from '@angular/router';
```

Comment:
It seems like you ran prettier on this file - can you add the pragma at the top so it's automatically checked going forward?